Brand/NCBrand.swift

```diff
@@ -68,7 +68,7 @@ let userAgent: String = {
     @objc public var use_themingColor: Bool = true
     @objc public var use_themingLogo: Bool = false
     @objc public var use_storeLocalAutoUploadAll: Bool = false
-    @objc public var use_loginflowv2: Bool = false                                              // Don't touch me !!
+    @objc public var use_loginflowv2: Bool = true                                              // Don't touch me !!
```

Comment:
@marinofaggiana The plan is to use v2 by default from now on, and v1 will be obsolete. Maybe we can remove this? Not sure if the brander will break

Reply:
Is it ok to remove it?